docs/book/src/plugins/kustomize-v2-alpha.md

```diff
@@ -107,6 +108,7 @@ The following scaffolds will be created or updated by this plugin:
 * Check the kustomize [plugin implementation](https://github.com/kubernetes-sigs/kubebuilder/tree/master/pkg/plugins/common/kustomize)
 * Check the [kustomize documentation][kustomize-docs]
 * Check the [kustomize repository][kustomize-github]
+* Check the [kustomize release note](https://github.com/kubernetes-sigs/kustomize/releases)
```

Comment:
We need to add the link for kustomize v4.0.0 specific release to let people be able to check the breaking changes and also for v5.0.0 one because of the same reasons.

Reply:
v4: https://github.com/kubernetes-sigs/kustomize/releases/tag/kustomize%2Fv4.5.7
v5: https://github.com/kubernetes-sigs/kustomize/releases/tag/kustomize%2Fv5.0.0
I want you to just confirm that these are the links you are talking about.

Reply:
Breaking changes are introduced usually only in MAJOR releases (see: https://semver.org/)
So, it is v4.0.0 (I think we have the link for this one can you please check in the page, if not we need to add ) and 5.0.0

Reply:
I tried to check the file but it is giving me 404 error